config/locales/devise.en.yml

```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
```

Comment:
Was this added as part of the devise generator? 99% of it seems irrelevant is why I ask, but if it was generated I'm fine with it(ish).

Reply:
Yup, that's a generated file. Happy to remove or trim it if you think it will be confusing. (We don't use it in ETD.)

Reply:
Nah, it's fine. Thanks for confirming though.